sdk/resourcemanager/Azure.ResourceManager/src/Azure.ResourceManager.csproj

```diff
@@ -10,13 +10,18 @@
     <UseDefaultNamespaceAndOutputFolder>false</UseDefaultNamespaceAndOutputFolder>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <AzureCoreSharedSources>$(MSBuildThisFileDirectory)/../../../../sdk/core/Azure.Core/src/Shared/</AzureCoreSharedSources>
```

Comment:
Because the build targets for ARM libraries still live in the autorest.csharp package and point to the shared source files there.  This PR attempts to point to the shared source files in azure-sdk-for-net, and can't do it updating the value of the `AzureCoreSharedSources` variable.